AutoClicker/Views/SettingsWindow.xaml.cs

```diff
@@ -100,6 +115,18 @@ private void StopKeyTextBox_KeyDown(object sender, KeyEventArgs e)
             e.Handled = true;
             SelectedStopKey = newKeyMapping;
         }
+        private void toggleKeyTextBox_KeyDown(object sender, KeyEventArgs e)
+        {
+            KeyMapping newKeyMapping = GetNewKeyMapping(e.Key);

```

Comment:
these three KeyDown methods look pretty similar, maybe we should extract the common code to a method

Reply:
@oriash93
- The UI should be updated with your suggestions. 
- The buttons update only when the bindings update. If you hover over the buttons they update. 
- I don't think we should because there may be users with very high resolution displays and the min/max values will have drastically different effects on their experience than on users with low resolution displays

Reply:
I added a generic method for setting the hotkey on keydown